GOVERNANCE.md

```diff
@@ -115,4 +115,9 @@ To create a new release:
     `git log --pretty=format:%s master...v0.1`.
     Just replace "v0.1" with the name of the previous release.
   * Press `Publish release` button
+* Create an "announcement" highlighting the key features of the new release
+  and any potential noteworthy activities of the group:
+  * Send it to the mailing list
+  * Add it to the "announcement" section of our
+    [website](https://cloudevents.io/)
```

Comment:
I'll update this link to the announcement section on it's created - @barkerd427 is working on this.